README.md

```diff
@@ -39,6 +39,17 @@ npm i puppeteer
 
 Note: When you install Puppeteer, it downloads a recent version of Chromium (~170Mb Mac, ~282Mb Linux, ~280Mb Win) that is guaranteed to work with the API. To skip the download, see [Environment variables](https://github.com/GoogleChrome/puppeteer/blob/v1.7.0/docs/api.md#environment-variables).
 
+
+### puppeteer-core
+
+Since version 1.7.0 we publish a [`puppeteer-core`](https://www.npmjs.com/package/puppeteer-core) package: puppeteer that doesn't download Chromium by default.
```

Comment:
```md
Since version 1.7.0 we publish the [`puppeteer-core`](https://www.npmjs.com/package/puppeteer-core) package, a version of Puppeteer that doesn't download Chromium by default.
```

Reply:
Done.